LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The Oscar.jl package is licensed under the GNU Public License, Version 3.0+:
 
-> Copyright (c) 2019 The OSCAR Development Team.
+> Copyright (c) 2019-2024 [The OSCAR Development Team](https://github.com/oscar-system/Oscar.jl/graphs/contributors)
```

Comment:
https://www.oscar-system.org/contributors/

Reply:
The year should be updated.

For legal purposes it would be important to "define" what "The OSCAR Development Team" means. A link to a list of contributors would be a good start but it is a bit vague.

In GAP we "solved" this by including an explicit explanation of what the "team" is, and in this case, a full list of people. See https://github.com/gap-system/gap/blob/master/COPYRIGHT . Note that we wrote "which include" on purpose to make it open to "we may have missed a few people". I think the GAP text could be even better if it explicitly mentioned that one should also check the `git log` or something.

So in our case that could be done by having an explicit text, like this:
```suggestion
> Copyright (c) 2019-2024 The OSCAR Development Team
>
> See <https://www.oscar-system.org/contributors/> for a list of OSCAR
> Development Team members. Note that this list may occasionally be slightly
> outdated. Additional contributors may be found in the `git log` of the main
> OSCAR repository and other related repositories.
```